### ISSUE TITLE
[Bug]: e2e tests are failing for unknown reasons

### ISSUE THREAD

**Issue by bautistaaa** [typehero/typehero#1364]
### What happened?

The a11y test started failing after tweaking some unrelated html

commit: https://github.com/typehero/typehero/commit/e46a4ffb4b9fbe32583222973cca54a8625f2683

**Comment by mdwiltfong** [comment]
Hey @bautistaaa ! 

Let me know what you think of that PR!
